src/main/java/uk/gov/hmcts/ccd/domain/model/casedataaccesscontrol/matcher/GrantTypeMatcher.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol.matcher;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol.RoleAssignment;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseTypeDefinition;
+
+@Slf4j
```

Comment:
Add in the debug logging to stay consistent with others? Or remove this. (Same for other matcher)

Reply:
done